### PR TITLE
Add ObjectDict class

### PR DESCRIPTION
**Pull request by iver56**

### CODE REVIEW

File: torch_audiomentations/utils/object_dict.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Inspired by tornado
+# https://www.tornadoweb.org/en/stable/_modules/tornado/util.html#ObjectDict
+
+try:
+    import typing
+    from typing import cast
```
Comment (hbredin) [inline torch_audiomentations/utils/object_dict.py:6]: Can you explain why this is imported? 
Is this a way to detect whether one can use `typing.Dict[...]` syntax?

Comment (iver56) [reply torch_audiomentations/utils/object_dict.py:6]: Hmm, I'm not sure. I think it is a copy of tornado code from a time when it supported Python 3.4. This try except is probably not needed anymore, but it also doesn't hurt too much. Should I remove it?

Comment (hbredin) [reply torch_audiomentations/utils/object_dict.py:6]: I would remove it, yes.
Note that I cherry-picked this PR in #123.

Comment (iver56) [reply torch_audiomentations/utils/object_dict.py]: Done

**Review by hbredin — APPROVED** [review]

**Comment by iver56** [comment]
Should I merge it, or will you cherry-pick it into your branch?

**Comment by hbredin** [comment]
I cherry-picked it.

**Pull request closed without merging**